Description: Don't install LICENSE, require debian-menu
Author: Julien Danjou <acid@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- awesome-3.4.11.orig/CMakeLists.txt
+++ awesome-3.4.11/CMakeLists.txt
@@ -32,8 +32,7 @@ set(AWE_THEMES_DIR ${SOURCE_DIR}/themes)
 set(AWE_DOC_FILES
     ${SOURCE_DIR}/AUTHORS
     ${SOURCE_DIR}/BUGS
-    ${SOURCE_DIR}/README
-    ${SOURCE_DIR}/LICENSE)
+    ${SOURCE_DIR}/README)
 
 set(AWE_SRCS
     ${SOURCE_DIR}/awesome.c
--- awesome-3.4.11.orig/awesomerc.lua.in
+++ awesome-3.4.11/awesomerc.lua.in
@@ -7,6 +7,9 @@ require("beautiful")
 -- Notification library
 require("naughty")
 
+-- Load Debian menu entries
+require("debian.menu")
+
 -- {{{ Error handling
 -- Check if awesome encountered an error during startup and fell back to
 -- another config (This code will only ever execute for the fallback config)
@@ -37,8 +40,8 @@ end
 beautiful.init("@AWESOME_THEMES_PATH@/default/theme.lua")
 
 -- This is used later as the default terminal and editor to run.
-terminal = "xterm"
-editor = os.getenv("EDITOR") or "nano"
+terminal = "x-terminal-emulator"
+editor = os.getenv("EDITOR") or "editor"
 editor_cmd = terminal .. " -e " .. editor
 
 -- Default modkey.
@@ -85,6 +88,7 @@ myawesomemenu = {
 }
 
 mymainmenu = awful.menu({ items = { { "awesome", myawesomemenu, beautiful.awesome_icon },
+                                    { "Debian", debian.menu.Debian_menu.Debian },
                                     { "open terminal", terminal }
                                   }
                         })
